From: DanilChapovalov <danilchap@webrtc.org>
Date: Fri, 29 Sep 2023 18:43:57 +0200
Subject: [Build] Add missing absl includes (#34499)

To avoid depending on transitive includes, specially avoid relying on
transitive include status.h -> str_cat.h that is removed in the latest
version of abseil

Origin: upstream, 784e3b1c40fff4cb1381ddc6b9917b29bafab8f2
---
 src/core/lib/iomgr/tcp_posix.cc | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/core/lib/iomgr/tcp_posix.cc b/src/core/lib/iomgr/tcp_posix.cc
index 72e1b66..769c15f 100644
--- a/src/core/lib/iomgr/tcp_posix.cc
+++ b/src/core/lib/iomgr/tcp_posix.cc
@@ -20,6 +20,12 @@
 
 #include <grpc/impl/codegen/grpc_types.h>
 
+#include "absl/base/thread_annotations.h"
+#include "absl/status/status.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/string_view.h"
+#include "absl/types/optional.h"
+
 #include "src/core/lib/gprpp/global_config_generic.h"
 #include "src/core/lib/iomgr/exec_ctx.h"
 #include "src/core/lib/iomgr/port.h"
